lib/reporting/authentication_report.rb

```diff
@@ -170,7 +170,7 @@ def query
         fields
             name
           , properties.user_id AS user_id
-        | filter properties.service_provider = %{issuer}
+        | filter properties.service_provider IN %{issuers}
```

Comment:
Just confirming: Does using an array here just intrinsically work? Without looking through the parser code I would have guessed we had to `.join(', ')` here. I'm inferring (based on tests) that it gets evaluated by a smarter parser.

Reply:
Good question! It's the call to `quote()` above that formats it as an array